packages/mapviewer/src/modules/menu/components/header/AdditionalInfoCollapsable.vue

```diff
@@ -0,0 +1,58 @@
+<script setup>
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { ref } from 'vue'
+
+import MoreInfo from '@/modules/menu/components/help/MoreInfo.vue'
+import UpdateInfo from '@/modules/menu/components/help/UpdateInfo.vue'
+
+const isOpen = ref(false)
+</script>
+
+<template>
+    <div
+        class="z-menu-1-on-hover"
+        @mouseenter="isOpen = true"
+        @mouseleave="isOpen = false"
+        @click="isOpen = true"
+    >
+        <!-- Menu Button -->
+        <button class="btn m-0 px-1 btn-xs darken-on-hover no-bottom-rounded-corner">
+            <FontAwesomeIcon :icon="isOpen ? 'caret-up' : 'caret-down'" />
+            <span class="ms-1">News</span>
+        </button>
+
+        <!-- Collapsible Menu Items -->
+        <div
+            v-if="isOpen"
+            class="d-flex flex-column-reverse shadow-lg position-absolute"
+            data-cy="news-menu"
+        >
+            <MoreInfo class="darken-on-hover collapsable-menu-element px-1 rounded-0 z-3" />
+            <UpdateInfo class="darken-on-hover collapsable-menu-element px-1 rounded-0 z-3" />
+        </div>
+    </div>
+</template>
+
+<style lang="scss" scoped>
+@import '@/scss/webmapviewer-bootstrap-theme';
+// we need this component to be under the searchbar when not hovered, and over it when hovered.
+.z-menu-1-on-hover {
+    z-index: $zindex-menu - 1;
+}
+.z-menu-1-on-hover:hover {
+    z-index: $zindex-menu + 1;
+}
+.darken-on-hover {
+    background-color: $nav-tabs-link-active-color;
+}
+.darken-on-hover:hover {
+    background-color: $list-item-hover-bg-color;
+}
+.no-bottom-rounded-corner {
+    border-bottom-left-radius: 0px;
+    border-bottom-right-radius: 0px;
+}
```

Comment:
please use some [bootstrap class](https://getbootstrap.com/docs/5.3/utilities/borders/#radius) instead

Reply:
I did not find them when I was searching for them. Sorry I'm blind.